docs/dev.rst

```diff
@@ -1,20 +1,540 @@
+
+    Example: ``namespace_strip: npsp``
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - zip_url: https://github.com/SalesforceFoundation/CumulusReports/archive/main.zip
+              subfolder: CumulusReports-main/src
+              namespace_strip: npsp
+
+..
+
+    When ``update_dependencies`` runs, the zip is retrieved and the string ``npsp__`` is stripped from all files and filenames in the zip before deployment.  This option is most useful when setting up an unmanaged development environment for an extension package that normally uses managed dependencies.
+    
+    This example takes the NPSP Reports & Dashboards project's unmanaged metadata and strips the references to ``npsp__`` to deploy it against an unmanaged version of NPSP.
+
+
+GitHub Repository Dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+GitHub repository dependencies create a dynamic dependency between the current project and another CumulusCI project on GitHub.
+
+    Example: Salesforce EDA
+
+.. code-block:: yaml
+ 
+    project:
+        dependencies:
+            - github: https://github.com/SalesforceFoundation/EDA
+
+When ``update_dependencies`` runs, these steps are taken against the referenced repository.
+
+* Look for the ``cumulusci.yml`` file and parse if found.
+* Determine if the project has subfolders under ``unpackaged/pre``.  If found, deploy them first.
+* Determine if the project specifies any dependencies in the ``cumulusci.yml`` file.  If found, recursively resolve those dependencies and any dependencies belonging to them.
+* Determine whether to install the project as as a managed package or unmanaged metadata:
+    * If the project has a namespace configured in the ``cumulusci.yml`` file, treat the project as a managed package unless the unmanaged option is ``True``.
+    * If the project has a namespace and is *not* configured as unmanaged, use the GitHub API to locate the latest managed release of the project and install it.
+* If the project is an unmanaged dependency, the ``src`` or ``force-app`` directory is deployed.
+* Determine if the project has subfolders under ``unpackaged/post``. If found, deploy them next. Namespace tokens are replaced with ``namespace__`` if the project is being installed as a managed package, or an empty string otherwise.
+
+
+
+Reference Unmanaged Projects
+****************************
+
+If the referenced repository does not have a namespace configured, or if the dependency specifies the ``unmanaged`` option as ``True``, the repository is treated as an unmanaged repository.
+
+    Example: Salesforce EDA
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - github: https://github.com/SalesforceFoundation/EDA
+              unmanaged: True
+
+..
+
+    The EDA repository is configured for a namespace, but the dependency  specifies ``unmanaged: True``, so the dependency deploys unmanaged EDA and its dependencies.
```

Comment:
Second half of this sentence is a bit confusing. Is there a better way to say it? 

“...So EDA and its dependencies deploy/are deployed as unmanaged”?

Reply:
"... so EDA and its dependencies deploy as unmanaged metadata."